src/machi_pb_high_client.erl

```diff
@@ -58,68 +58,110 @@
           count=0     :: non_neg_integer()
          }).
 
+%% @doc official error types that is specific in Machi
+-type error_atoms() :: bad_arg | wedged | bad_checksum |
```

Comment:
It seems some built-in types are defined in singular, e.g. `atom()`, `term()`, `boolean()`. Which is better...?